README.md

```diff
@@ -370,7 +370,7 @@ To build a browser version from scratch when developing::
  $ yarn build
 ```
 
-To run tests (Jasmine)::
+To run tests (Jest)::
```

Comment:
Might be worth commenting that for the Browserify tests to run you must first follow `To build a browser version from scratch when developing` above

Reply:
Also the double `::` look like mistakes :P